obspy/scripts/runtests.py

```diff
@@ -45,24 +45,39 @@
     GNU Lesser General Public License, Version 3
     (https://www.gnu.org/copyleft/lesser.html)
 """
+import contextlib
+import os
 import sys
 from pathlib import Path
 
-import pkg_resources
-import obspy
-from obspy.core.util.misc import change_directory
-
 
 # URL to upload json report
 REPORT_URL = "tests.obspy.org"
 
 
+@contextlib.contextmanager
+def change_directory(path):
```

Comment:
Why do you prefer this function here? It may be generally useful to other functions/test suites then we will have to move it back.